test/01-msg.js

```diff
@@ -8,19 +8,21 @@ const bfe = require('../')
 tape('01 msg type', function (t) {
   const values = [
     '%HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK/pijYII=.sha256', // classic
-    'ssb:message/bendybutt-v1/HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK_pijYII=', // bendy-butt
     'ssb:message/gabbygrove-v1/QibgMEFVrupoOpiILKVoNXnhzdVQVZf7dkmL9MSXO5g=', // gabby-grove
+    '%HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK/pijYII=.cloaked', // cloaked
+    'ssb:message/bendybutt-v1/HZVnEzm0NgoSVfG0Hx4gMFbMMHhFvhJsG2zK_pijYII=', // bendy-butt
     'ssb:message/buttwoo-v1/QibgMEFVrupoOpiILKVoNXnhzdVQVZf7dkmL9MSXO5g=', // butt2
     'ssb:message/indexed-v1/QibgMEFVrupoOpiILKVoNXnhzdVQVZf7dkmL9MSXO5g=', // indexed-v1
   ]
 
   const encoded = bfe.encode(values)
 
   t.deepEquals(encoded[0].slice(0, 2), Buffer.from([1, 0]), 'classic msg')
-  t.deepEquals(encoded[1].slice(0, 2), Buffer.from([1, 4]), 'bendy butt msg')
-  t.deepEquals(encoded[2].slice(0, 2), Buffer.from([1, 1]), 'gabby grove msg')
-  t.deepEquals(encoded[3].slice(0, 2), Buffer.from([1, 5]), 'buttwoo msg')
-  t.deepEquals(encoded[4].slice(0, 2), Buffer.from([1, 6]), 'index msg')
+  t.deepEquals(encoded[1].slice(0, 2), Buffer.from([1, 1]), 'gabby grove msg')
+  t.deepEquals(encoded[2].slice(0, 2), Buffer.from([1, 2]), 'cloaked msg')
```

Comment:
Put these in same order as their index in the `bse.json`
Added test for `cloaked`